js/client-battle.js

```diff
@@ -11,7 +11,7 @@
 
 			this.isSideRoom = Tools.prefs('rightpanelbattles');
 
-			this.$el.addClass('ps-room-opaque').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
+			this.$el.addClass('ps-room-opaque').attr('role', 'main').html('<div class="battle">Battle is here</div><div class="foehint"></div><div class="battle-log" role="complementary" title="Battle Log"></div><div class="battle-log-add">Connecting...</div><div class="battle-controls" role="complementary" title="Battle Controls"></div><button class="battle-chat-toggle button" name="showChat"><i class="fa fa-caret-left"></i> Chat</button>');
```

Comment:
Also, `role="complementary" title="Battle Log"` is probably unnecessary here. It might conflict with `role="log" title="Battle Log"`. At least one of those should have a different label title?